demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -104,6 +109,18 @@ def __init__(
                                                    print_as_warnings=self.print_ignored_errors,
                                                    id_set_file=self.id_set_file) \
             if validate_id_set else None
+
+        try:
+            self.git_util = GitUtil(repo=Content.git())
+            self.branch_name = self.git_util.get_current_working_branch()
+        except InvalidGitRepositoryError:
+            if self.use_git:
+                raise
+            else:
+                self.git_util = None  # type: ignore
+                self.branch_name = ''
+                pass
```

Comment:
how is this ok? can you explain how you will be able to execute `self.get_changed_files_from_git()` in this case?
an inline comment in the code will help.

Reply:
Note - if the exception occurs and we use git - we raise it so it will never get to `self.get_changed_files_from_git()`

Anyway I'll add a comment.